codefresh.yml

```diff
@@ -0,0 +1,41 @@
+version: '1.0'
+
+steps:
+
+  agent_test:
```

Comment:
That's purposeful. It's an ['untyped' build step](https://docs.codefresh.io/docs/steps)